sci-libs/symengine/symengine-0.1.0.ebuild

```diff
@@ -0,0 +1,46 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+
+PYTHON_COMPAT=( python2_7 )
+PYTHON_REQ_USE=""
+
+inherit cmake-utils python-single-r1
+
+DESCRIPTION="Fast symbolic manipulation library, written in C++"
+HOMEPAGE="https://github.com/sympy/symengine"
+SRC_URI="https://github.com/sympy/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="python threads openmp"
+
+DEPEND="
+	dev-libs/jemalloc
+	dev-libs/boost
```

Comment:
On 08/25/15 09:26, Michael Schubert wrote:
> In sci-libs/symengine/symengine-0.1.0.ebuild
> <https://github.com/gentoo-science/sci/pull/466#discussion_r37806930>:
> 
>> +PYTHON_REQ_USE=""
>> +
>> +inherit cmake-utils python-single-r1
>> +
>> +DESCRIPTION="Fast symbolic manipulation library, written in C++"
>> +HOMEPAGE="https://github.com/sympy/symengine"
>> +SRC_URI="https://github.com/sympy/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
>> +
>> +LICENSE="MIT"
>> +SLOT="0"
>> +KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
>> +IUSE="python threads openmp"
>> +
>> +DEPEND="
>> +	dev-libs/jemalloc
>> +	dev-libs/boost
> 
> I thought that |python-single-r1| will just specify which versions the
> ebuild is compatible with and then the user change choose using the
> |USE=python_single_target_python*| flag - but I was wrong, with
> |python-single-r1| I can only have one python version in
> |${PYTHON_COMPAT}|. I don't see how I can easily set "python2_7 OR
> python3_3", because then I could use the cmake flag |DPYTHON_BIN=python*|.
> 

As I said I requested upstream to separate the python bindings and they
agreed that it was a good idea that they will act upon. So this is just
temporary.

For the rest read carefully
https://devmanual.gentoo.org/eclass-reference/python-single-r1.eclass/index.html
and https://wiki.gentoo.org/wiki/Project:Python/python-single-r1 you
probably forgot to use
python-single-r1_pkg_setup.

I was kind of planning to make the ebuild but you are doing a good job
given the state of upstream.

Francois